tests/foreman/ui/test_bookmark.py

```diff
@@ -43,11 +42,6 @@ def setUpClass(cls):
         for testing.
         """
         super(BookmarkTestCase, cls).setUpClass()
-        cls.per_page = entities.Setting().search(
-            query={'search': 'name="entries_per_page"'})[0]
-        cls.saved_per_page = str(cls.per_page.value)
-        cls.per_page.value = '100000'
```

Comment:
that was pretty necessary setting to avoid having pagination during testing. Did you verify all test cases on a server with a lot of data?